vehicle-model-lifecycle/src/download_vspec.py

```diff
@@ -84,21 +87,104 @@ def download_file(uri: str, local_file_path: str):
                 outfile.write(chunk)
 
 
-def get_vehicle_model_src():
-    manifest_data_str = require_env("VELOCITAS_APP_MANIFEST")
-    manifest_data = json.loads(manifest_data_str)
+def is_legacy_app_manifest(app_manifest_dict: Dict[str, Any]) -> bool:
+    """Check if the used app manifest is a legacy file.
+
+    Args:
+        app_manifest_dict (Dict[str, Any]): The app manifest.
+
+    Returns:
+        bool: True if the app manifest is a legacy file, False if not.
+    """
+    return "manifestVersion" not in app_manifest_dict
```

Comment:
should we also add the other versions? I know it most probably won't happen but for completeness maybe..

Reply:
We don't really have other versions we could distinguish. Maybe we could check the availability of fields to deduct the version  of the manifest?

Reply:
If we now have v3 we do have also v1 and v2 but thats what I meant, most probably nobody will ever write a manifest with manifestVersion: v1...

lets keep it as is and don't waste much time on that.